cadence/contracts/FindRegistry.cdc

```diff
@@ -0,0 +1,210 @@
+// FindRegistry 
+// - A contract that allows for the creation of registries
+// - Registries are used to store a list of accounts for a UUID and TTL
```

Comment:
can we add a comment here about the semantics of the design, like create a resource store it in a path set the TTL to the amount of blocks that should stop a user from a duplicated claim.